Parsers/Dynamic standup command.js

```diff
@@ -0,0 +1,22 @@
+/*
+activation_example: !standup
+regex: !standup
```

Comment:
Again. This is not following contributing guideline templates. The leading space would cause errors